src/frontend/js/media-management/library.tsx

```diff
@@ -237,17 +247,24 @@ export default function Library({
           onClick={() => setFileIndex(null)}
```

Comment:
Could you add the class `relative` as suggested [here](https://github.com/Integreat/integreat-cms/pull/909#discussion_r698574827) to the div in line 246? I can't make an automated suggestion because it's too far away from your changes.
Otherwise, the spinner is a bit off the center (which shows especially on small screens):
![Screenshot 2021-09-16 at 02-35-10 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/133530921-876045b3-9776-45bb-9ae5-eacd949d00ae.png)